release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -179,6 +179,12 @@ For more information, see the "Installation configuration parameters" section of
 The documentation for disabling the Bare Metal Operator is currently unavailable.
 ====
 
+[id="ocp-4-11-aws-marketplace"]
+==== AWS Marketplace offering
+{product-title} is now available on the AWS Marketplace.The AWS Marketplace offering is available to customers who procure {product-title} in North America.
+
+For more information, see link:https://access.redhat.com/articles/6675791[Installing OpenShift using AWS Marketplace].
+
```

Comment:
EMEA will follow shortly. Otherwise /lgtm

Reply:
Thanks @dmesser. I cannot add a future statement to the doc. However, as part of the work I will be doing in the help, I propose a section that details where the AWS (and Azure) marketplace offerings are available. This section can be updated as availability grows. WDYT?

Reply:
> I cannot add a future statement to the doc. However, as part of the work I will be doing in the help, I propose a section that details where the AWS (and Azure) marketplace offerings are available. This section can be updated as availability grows. WDYT?

Confirmed with @dmesser in Slack that this approach is OK. PR is approved.